spec/models/user_spec.rb

```diff
@@ -794,13 +794,44 @@
         FactoryBot.create(:user)
       }
 
-      context "with autofollow sharing enabled" do
-        it "should start sharing with autofollow account" do
+      context "when enabled" do
+        before do
           AppConfig.settings.autofollow_on_join = true
+        end
+
+        it "should start sharing with autofollow on join user" do
           person = FactoryBot.build(:person)
           AppConfig.settings.autofollow_on_join_user = person.diaspora_handle
 
-          expect(Person).to receive(:find_or_fetch_by_identifier).with(person.diaspora_handle).and_return(person)
+          expect(Person).to
+          receive(:find_or_fetch_by_identifier).with(person.diaspora_handle).and_return(person)
```

Comment:
This test is broken, as it expects the behavior I described above. It wants that only the `autofollow_on_join_user` (`person`) is followed, but since `autofollow_on_join_accounts` has a default value of the HQ account, it also tries to follow that one, which isn't expected by this test. So either change the behavior that when `autofollow_on_join_user` is set, it ignores the value of `autofollow_on_join_accounts` (which always has a default value set), or change the test, to also expect the default value of `autofollow_on_join_accounts`.

Also there is a syntax error with the `receive` on it's own line.